locales/en.json

```diff
@@ -71,28 +71,28 @@
 	"Header": "Header",
 	"Unordered List": "Unordered List",
 	"Ordered List": "Ordered List",
-	"Todo List": "Todo List",
+	"Todo List": "Checklist",
 	"Blockquote": "Blockquote",
 	"Bold font": "Bold font",
 	"Italics font": "Italics font",
 	"Strikethrough": "Strikethrough",
 	"Inserted text": "Inserted text",
-	"Marked text": "Marked text",
+	"Marked text": "Highlighted text",
```

Comment:
I've never seen the word "mark" to mean "highlighted". I think they use "marked" to refer the application of the `<mark>` tag. As the end user would not know what tag is behind, it would not make sense. And it's only `<mark>` to comply with the W3C standard that tags specify the semantics and not styling of text. (`<strike>` was superseded for this reason by `<del>`)